documentation/docs/identus/cloud-agent/environment-variables.md

```diff
@@ -59,8 +59,8 @@ The following enviroment variables can be used to configure Prism Agent:
 | DEFAULT_WALLET_AUTH_API_KEY                | The authentication API key to be used for default entity that uses default wallet.                                                                                 | String                | default                                      |
 | KEYCLOAK_ENABLED                           | Whether or not to enable Keycloak authentication and authorisation.                                                                                                | Boolean               | false                                        |
 | KEYCLOAK_URL                               | The Keycloak server URL.                                                                                                                                           | String                | http://localhost:9980                        |
-| KEYCLOAK_REALM                             | The Keycloak realm name.                                                                                                                                           | String                | `atala-demo`                                 |
-| KEYCLOAK_CLIENT_ID                         | The Keycloak client ID.                                                                                                                                            | String                | `prism-agent`                                |
-| KEYCLOAK_CLIENT_SECRET                     | The Keycloak client secret.                                                                                                                                        | String                | `prism-agent-demo-secret`                    |
+| KEYCLOAK_REALM                             | The Keycloak realm name.                                                                                                                                           | String                | `identus-demo`                                 |
+| KEYCLOAK_CLIENT_ID                         | The Keycloak client ID.                                                                                                                                            | String                | `cloud-agent`                                |
```

Comment:
Is it changed in the identus-cloud-agent repo as well?

Reply:
See answer above, In the mean time should I revert it?